wp-content/plugins/jetpack/_inc/build/carousel/swiper-bundle.min.js

```diff
@@ -0,0 +1,2 @@
+/*! For license information please see swiper-bundle.min.js.LICENSE.txt */
```

Comment:
:warning: **Warning( severity 5 )**: File appears to be minified and cannot be processed (*Internal.Tokenizer.Exception*).